requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.8.0.dev20160718123552
+coala>=0.8.0.dev20160729111621
```

Comment:
erm, actually the revert of that, https://github.com/coala-analyzer/coala/commit/ac2d5362488bb59c8137f99251c6941889f48e1e